tests/Php70/Php70Test.php

```diff
@@ -34,18 +34,24 @@ public function provideIntdiv()
     }
 
     /**
-     * @expectedException ArithmeticError
+     * @expectedException \ArithmeticError
      */
     public function testIntdivArithmetic()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('HHVM fails here.');
```

Comment:
Removing HHVM really makes sense as just skipping tests that fail for it just means we don't support it anyway.